shared/ui/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,49 @@
+import { ComponentPropsWithoutRef } from 'react';
+
+import { Typography } from '@/shared/ui';
+import * as Radio from '@radix-ui/react-radio-group';
+import { clsx } from 'clsx';
+
+import style from './radio-group.module.scss';
+
+export type OptionsRadioGroup = {
+  disabled?: boolean;
+  id?: string;
+  label?: string;
+  value: string;
+}[];
+
+export type RadioGroupProps = {
+  className?: string;
```

Comment:
Redundant. `RadioGroupProps` already has `className` with exactly the same value.